coverage-tests.js

```diff
@@ -1080,196 +1080,178 @@ test('should send dom and location when check window', {
-    const result = eyes.close(false).ref('result')
-    const info = helpers.getTestInfo(result).ref('info')
+    const result = eyes.close(false)
+    const info = helpers.getTestInfo(result)
     assert.equal(info.actualAppOutput[0].image.location, {x: 10, y: 0})
     assert.equal(info.actualAppOutput[0].image.hasDom, true)
-    const dom = helpers.getDom(result, info.actualAppOutput[0].image.domId).ref('dom')
-    const targetElement = dom.getNodesByAttribute('data-applitools-target').ref('targetElement')
-    assert.equal(targetElement.length, 1)
-    assert.equal(targetElement[0].rect.top, -2) // not 0 because of 2px border
-    assert.equal(targetElement[0].rect.left, 8) // not 10 because of 2px border
-    const scrollingElements = dom.getNodesByAttribute('data-applitools-scroll').ref('scrollingElements')
+    const dom = helpers.getDom(result, info.actualAppOutput[0].image.domId)
+    const targetElement = dom.getNodesByAttribute('data-expected-target')
+    assert.equal(info.actualAppOutput[0].image.location.x, helpers.math.round(targetElement[0].rect.left + 2))
+    assert.equal(info.actualAppOutput[0].image.location.y, helpers.math.round(targetElement[0].rect.top + 2))
+    const scrollingElements = dom.getNodesByAttribute('data-applitools-scroll')
     assert.equal(scrollingElements.length, 1)
     assert.equal(scrollingElements[0].attributes['data-applitools-scroll'], 'true')
-    assert.equal(scrollingElements[0].attributes['data-applitools-expected-scroll'], 'true')
+    assert.equal(scrollingElements[0].attributes['data-expected-scroll'], 'true')
   }
 })
 
 test('should send dom and location when check region by selector in frame', {
   page: 'Default',
   variants: {
     '': {vg: false},
-    'with vg': {vg: true},
+    'with vg': {vg: true,skipEmit: true},
   },
   test({driver, eyes, assert, helpers}) {
     eyes.open({appName: 'Applitools Eyes SDK', viewportSize})
-    driver.executeScript('window.scrollTo(0, 350)')
-    const frameElement = driver.findElement('[name="frame1"]').ref("frameElement")
+    const frameElement = driver.findElement('[name="frame1"]')
     driver.executeScript('arguments[0].setAttribute("data-applitools-frame", "true");', frameElement)
-    driver.executeScript(`arguments[0].contentDocument.querySelector('[name="frame1-1"]').setAttribute("data-applitools-target", "true");`, frameElement)
+    driver.executeScript(`arguments[0].contentDocument.querySelector('[name="frame1-1"]').setAttribute("data-expected-target", "true");`, frameElement)
     eyes.check({frames: [frameElement], region: '[name="frame1-1"]'})
-    const result = eyes.close(false).ref('result')
-    const info = helpers.getTestInfo(result).ref('info')
-    assert.equal(info.actualAppOutput[0].image.location, {x: 106, y: 0})
+    const result = eyes.close(false)
+    const info = helpers.getTestInfo(result)
+    assert.equal(info.actualAppOutput[0].image.location, {x: 108, y: 0})
```

Comment:
I think this should be 106 because the border of `frame1-1` is included in the screenshot

Reply:
the border of the parent frame adds these 2 pixels.